charts/scalardl/templates/ledger/deployment.yaml

```diff
@@ -74,18 +82,23 @@ spec:
                 name: {{ include "scalardl.fullname" . }}-ledger
               {{- end }}
                 key: db-password
-          - name: SCALAR_DB_STORAGE
+          - name: HELM_SCALAR_DB_STORAGE
             value: "{{ .Values.ledger.scalarLedgerConfiguration.dbStorage }}"
           - name: SCALAR_DL_LEDGER_LOG_LEVEL
```

Comment:
We cannot add `HELM_` prefix to this environment variable name since this is used in the `log4j.properties` in the container and we cannot override `log4j.properties` using Helm Charts at this time.